### ISSUE TITLE
Default storageclass required

### ISSUE THREAD

**Issue by btat** [SUSE/doc-cap#813]
As per Rocketchat `confirmed that cap2 requires a storageclass set as default`. This will need to be added.

**Comment by viccuad** [comment]
Nowadays this is not true, one can configure a storageclass for kubecf, using in the values.yaml:

```
kube:
  storage_class: yourstorageclass
```

See https://github.com/cloudfoundry-incubator/kubecf/pull/153

**Comment by btat** [comment]
> 
> 
> Nowadays this is not true, one can configure a storageclass for kubecf, using in the values.yaml:
> 
> ```
> kube:
>   storage_class: yourstorageclass
> ```
> 
> See [cloudfoundry-incubator/kubecf#153](https://github.com/cloudfoundry-incubator/kubecf/pull/153)

Configuring a storage_class through the values.yaml is already mentioned in the docs. I _think_ the RC message in the description was referring to needing a storage class marked as default in the case one is not configured?

@HartS could you confirm/provide more context?

**Comment by HartS** [comment]
Not 100% sure if that's correct, as I was having issues unless I actually set a default storageclass.  @viccuad are you sure you were able to deploy kubecf without the same storageclass used *also* set to default? Because I did try the catapult option which should set that.

However, I've also been having other issues deploying CAP on CaaSP test clusters I've been using, so it's possible that at the time I was attributing those to not having a default storageclass set.
